catris.py

```diff
@@ -1469,6 +1472,7 @@ def render(self) -> None:
     async def _receive_bytes(self) -> bytes | None:
         # TODO: is the error handling needed?
```

Comment:
While running the fuzzer in this PR and a couple variations, I got the error occasionally.

```suggestion
```